UITests/Responses/TransferMethod/Individual/TransferMethodConfigurationKeysResponse.json

```diff
@@ -0,0 +1,215 @@
+                                                        "currency": "USD"
+                                                    }
+                                                ]
+                                            }
+                                        },
+                                        {
+                                            "code": "PAYPAL_ACCOUNT",
+                                            "name": "PayPal Account",
+                                            "fees": {
+                                                "nodes": [
+                                                    {
+                                                        "value": "0.25",
+                                                        "feeRateType": "FLAT",
+                                                        "currency": "USD"
+                                                    }
+                                                ]
+                                            }
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    }
+                },
+                {
+                    "code": "GB",
+                    "name": "UNITED KINGDOM",
+                    "currencies": {
+                        "nodes": [
+                            {
+                                "code": "EUR",
+                                "name": "EUR",
+                                "transferMethodTypes": {
+                                    "nodes": [
+                                        {
+                                            "code": "BANK_ACCOUNT",
+                                            "name": "Bank Account",
+                                            "processingTime" : "1-2 Business days",
+                                            "fees": {
+                                                "nodes": [
+                                                    {
+                                                        "value": "1.50",
+                                                        "feeRateType": "FLAT",
+                                                        "currency": "EUR"
+                                                    }
+                                                ]
+                                            }
+                                        }
+                                    ]
+                                }
+                            },
+                            {
+                                "code": "GBP",
+                                "name": "GBP",
+                                "transferMethodTypes": {
+                                    "nodes": [
+                                        {
+                                            "code": "BANK_ACCOUNT",
```

Comment:
This might be missing some data.  Please use the following instead:
```
{
  "data": {
    "countries": {
      "nodes": [
        {
          "code": "CA",
          "name": "Canada",
          "currencies": {
            "nodes": [
              {
                "code": "CAD",
                "name": "Canadian Dollar",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "1-2 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "CA",
                            "currency": "CAD",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "2.20",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    },
                    {
                      "code": "PAYPAL_ACCOUNT",
                      "name": "PayPal Account",
                      "processingTime": "IMMEDIATE",
                      "fees": {
                        "nodes": [
                          {
                            "country": "CA",
                            "currency": "CAD",
                            "transferMethodType": "PAYPAL_ACCOUNT",
                            "value": "0.25",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              },
              {
                "code": "USD",
                "name": "United States Dollar",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "1-2 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "CA",
                            "currency": "USD",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "2.00",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              }
            ]
          }
        },
        {
          "code": "HR",
          "name": "Croatia",
          "currencies": {
            "nodes": [
              {
                "code": "EUR",
                "name": "Euro",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "2-3 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "HR",
                            "currency": "EUR",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "1.50",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              },
              {
                "code": "HRK",
                "name": "Croatian Kuna",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "1-2 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "HR",
                            "currency": "HRK",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "1.50",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              }
            ]
          }
        },
        {
          "code": "MX",
          "name": "Mexico",
          "currencies": {
            "nodes": [
              {
                "code": "MXN",
                "name": "Mexican Peso",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "1-2 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "MX",
                            "currency": "MXN",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "26.15",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              }
            ]
          }
        },
        {
          "code": "GB",
          "name": "United Kingdom",
          "currencies": {
            "nodes": [
              {
                "code": "EUR",
                "name": "Euro",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "2-4 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "GB",
                            "currency": "EUR",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "1.50",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              },
              {
                "code": "GBP",
                "name": "British Pound",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "1-2 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "GB",
                            "currency": "GBP",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "1.20",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              }
            ]
          }
        },
        {
          "code": "US",
          "name": "United States",
          "currencies": {
            "nodes": [
              {
                "code": "USD",
                "name": "United States Dollar",
                "transferMethodTypes": {
                  "nodes": [
                    {
                      "code": "BANK_ACCOUNT",
                      "name": "Bank Account",
                      "processingTime": "1-2 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "US",
                            "currency": "USD",
                            "transferMethodType": "BANK_ACCOUNT",
                            "value": "2.00",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    },
                    {
                      "code": "BANK_CARD",
                      "name": "Debit Card",
                      "fees": {
                        "nodes": [
                          {
                            "country": "US",
                            "currency": "USD",
                            "transferMethodType": "BANK_CARD",
                            "value": "1.75",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    },
                    {
                      "code": "WIRE_ACCOUNT",
                      "name": "Wire Transfer",
                      "processingTime": "1-3 Business days",
                      "fees": {
                        "nodes": [
                          {
                            "country": "US",
                            "currency": "USD",
                            "processingTime": "1-3 Business days",
                            "transferMethodType": "WIRE_ACCOUNT",
                            "value": "20.00",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    },
                    {
                      "code": "PAYPAL_ACCOUNT",
                      "name": "PayPal Account",
                      "processingTime": "IMMEDIATE",
                      "fees": {
                        "nodes": [
                          {
                            "country": "US",
                            "currency": "USD",
                            "transferMethodType": "PAYPAL_ACCOUNT",
                            "value": "0.25",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }
                    }
                  ]
                }
              }
            ]
          }
        }
      ]
    }
  }
}
```